cli/tests/integration_tests.rs

```diff
@@ -1322,7 +1322,7 @@ fn repl_test_assign_underscore_error() {
   assert!(
     out.ends_with("Last thrown error is no longer saved to _error.\n1\n1\n")
   );
-  assert_eq!(err, "Thrown: 2\n");
+  assert_eq!(err, "Uncaught 2\n");
```

Comment:
Minor deviation from the current implementation.

Reply:
IIRC "Uncaught" I added to error message by V8 itself. I think this change is fine.